blog-migration.js

```diff
@@ -80,11 +125,13 @@ export async function main(index, cached, output, force) {
 
             const mdast = await getMdast(markdown);
             const tableMap = getTableMap(mdast);
-            const migrationBlocks = tableMap.filter((block) => Object.keys(MIGRATION).includes(block.blockName));
+            const tableBlocks = tableMap.map((block) => block.blockName)
+            const migrationMap = tableMap.filter((block) => Object.keys(MIGRATION).includes(block.blockName));
+            const migrationBlocks = migrationMap.map((block) => block.blockName);
 
-            if (migrationBlocks.length === 0 && !entry.includes(BANNERS_PATH)) {
-                console.log('Skipping', entry, tableMap.map(block => block.blockName));
-                report.skipped.push(entry);
+            if (migrationMap.length === 0 && !entry.includes(BANNERS_PATH)) {
+                console.log(`${i}/${entries.length} Skipping ${entry}`, tableMap.map(block => block.blockName));
```

Comment:
Isn't this the same as your variable?

```suggestion
                console.log(`${i}/${entries.length} Skipping ${entry}`, tableBlocks);
```

Reply:
Yes it is, thanks.